.github/workflows/release.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         with:
           node-version: '14'
```

Comment:
I've been moving this to `.nvm` files to make the switch to 16 easier, but good either way.